shell/browser/api/atom_api_web_request_ns.cc

```diff
@@ -99,6 +100,33 @@ bool MatchesFilterCondition(extensions::WebRequestInfo* info,
   return false;
 }
 
+// Convert HttpResponseHeaders to V8.
+//
+// Note that while we already have converters for HttpResponseHeaders, we can
+// not use it because it lowercases the header keys, while the webRequest has
+// to pass the original keys.
+v8::Local<v8::Value> HttpResponseHeadersToV8(
+    net::HttpResponseHeaders* headers) {
+  base::DictionaryValue response_headers;
+  if (headers) {
+    size_t iter = 0;
+    std::string key;
+    std::string value;
+    while (headers->EnumerateHeaderLines(&iter, &key, &value)) {
+      if (response_headers.FindKey(key)) {
+        base::ListValue* values = nullptr;
+        if (response_headers.GetList(key, &values))
+          values->AppendString(value);
+      } else {
+        auto values = std::make_unique<base::ListValue>();
+        values->AppendString(value);
```

Comment:
AppendString is deprecated, but I think we can pass a ListStorage into the ListValue ctor, so something like

```c++
auto values = std::make_unique<base::ListValue>({ value }); // not sure if this will compile?
auto values = std::make_unique<base::ListValue>(base::Value::ListStorage(value));
```